ser/train.py

```diff
@@ -0,0 +1,36 @@
+import torch
+# from ser.data import training_dataloader, validation_dataloader
+import torch.nn.functional as F
+
+def train(model, device, optimizer, epochs, training_dataloader, validation_dataloader):
```

Comment:
did you spot the inefficiency? Hint, it's somewhere in here :) Abstraction should help!